examples/imitation_learning_openbot/generate_dataset.py

```diff
@@ -258,7 +221,7 @@
         # get the updated compass observation (with the last recorded position set as goal)
         goal_position_xy = state_data[num_iterations-1][0:2]
         for i in range(num_iterations):
-            position_xy_current = np.array([state_data[i][0], state_data[i][1]], dtype=np.float32)
+            position_xy_current = np.array([state_data[i][0], state_data[i][1]], dtype=np.float64)
             yaw_current = state_data[i][4]
             compass_data[i] = get_compass_observation(goal_position_xy, position_xy_current, yaw_current)
 
```

Comment:
- write in this order: `ctrlLog.txt`, `rgbFrames.txt`, `goalLog.txt`, `debugLog.txt`
- combine all of the debug data into a single DataFrame

Reply:
Updated this